test/fixtures/shared/network.tf

```
@@ -26,20 +26,20 @@ resource "google_compute_network" "network" {
 }
 
 resource "google_compute_subnetwork" "subnetwork-a" {
-  name          = "cft-cloud-nat-test-${random_string.suffix.result}-a"
-  region        = "${var.region}"
-  network       = "${google_compute_network.network.self_link}"
+  name          = "cft-cloud-nat-test-${random_string.suffix.result}"
```

The suffix should not be dropped.

```suggestion
  name          = "cft-cloud-nat-test-${random_string.suffix.result}-a"
```